src/middleware/handled.ts

```diff
@@ -1,7 +1,7 @@
 import validator from "validator";
-import { Channel } from "../entities/Channel";
 import { User } from "../entities/User";
-import { ErrorCode, WestEggError } from "../errors";
+import { ErrorCode, WestEggError } from "@gatsby-tv/types";
+import { Channel } from "../entities/Channel";
 
 const HANDLE_MIN_LENGTH = 3;
```

Comment:
Can you make the minimum handle length 4? This way they don't conflict with the `api` route I have on the frontend. This is also the same length that both Twitch and Twitter requires as well.